templates/project/.php-cs-fixer.dist.php

```diff
@@ -6,6 +6,17 @@
  * It's auto-generated by sonata-project/dev-kit package.
  */
 
+declare(strict_types=1);
```

Comment:
In tests/bootstrap.php, we have the following order
<img width="864" alt="image" src="https://user-images.githubusercontent.com/9052536/159439699-3aebea2a-46b8-4fc7-8da7-f295ee08ba49.png">

- strict
- header
- uses
- DO NOT EDIT THIS FILE

In lot of other files, we have header before the `do not edit this file` message.

I would say we should always use the same order
- declare strict
- header
- Do not edit this file
- uses and php code

Reply:
Agree and fixed on every php file.